ol/miner/src/commands/start_cmd.rs

```diff
@@ -79,7 +81,7 @@ impl Runnable for StartCmd {
             match backlog::process_backlog(&cfg, &tx_params, is_operator) {
                 Ok(()) => status_ok!("Backlog:", "backlog committed to chain"),
                 Err(e) => {
-                    println!("Failed fetching remote state: {}", e);
+                    println!("WARN: Failed fetching remote state: {}", e);
```

Comment:
I thin this kind of logging with severity is very good and crucial both for dev/production for such a big code base, would be nice to use a logging library where we do not have to write "WARN" manually.